packages/web-styles/src/placeholders/_color.scss

```diff
@@ -3,20 +3,49 @@
 @use '../variables/color';
 @use '../functions/color' as color-fn;
 
-// CWF color variables not rendered by bootstrap
-%color-variables {
-  --text-color: #{color.$black};
-  --text-color-rgb: #{color-fn.rgb-values(color.$black)};
-  --background-color: #{color.$white};
-  --background-color-rgb: #{color-fn.rgb-values(color.$white)};
+// background rgb properties
+%color-background-rgb {
+  background-color: rgb(var(--bg-rgb)) !important;
```

Comment:
Is !important necessary here?

Reply:
No, it has been taken over from bootstrap.
However, without !important it can be more easily overwritten by a higher specificity.
Therefore I would prefer to leave this as it is.